numba/cuda/tests/cudapy/test_reduction.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import print_function
+import numpy as np
+from numba import cuda
+from numba import unittest_support as unittest
+#from numba.cuda import reduction
```

Comment:
Is this commented out because of a circular import? It sounds unlikely in this module.